pkg/image/apis/image/v1/helpers.go

```diff
@@ -113,10 +113,12 @@ func ImageWithMetadata(image *Image) error {
 	case 1:
 		image.DockerImageManifestMediaType = schema1.MediaTypeManifest
 
-		if len(manifest.History) == 0 {
```

Comment:
@legionus i'm still trying to figure out the problem with invalid manifest... unfortunately i haven't fixed it yet :/